# Problem Set 9

## Part II  

In [1]:
import numpy as np
import matplotlib.pyplot as plt
import matplotlib as mpl
import sympy as sp

### Problem 1  

$$ \mathbf{F}(x,y) = \big(y^3-6y\big)\,\mathbf{i} + \big(6x - x^3\big)\,\mathbf{j}$$  

**a)**  Using Green's theorem, find the simple close curve $C$ for which the integral $\displaystyle \oint_C\mathbf{F}\cdot\mathrm{d}\mathbf{r}$ (with positive oritentation) will have the largest positive value.  

$$ \mathrm{curl }\,\mathbf{F} = 6-3x^2 - 3y^2 + 6 = 12 - 3\big(x^2+y^2\big) $$

$$ \mathrm{curl }\,\mathbf{F} = 12 - 3r^2$$

Via Green's theorem,  

$$ \oint_C\mathbf{F}\cdot\mathrm{d}\mathbf{r} = \int\!\int_R 12\,\mathrm{d}A - 3\int\!\int_R r^2\,\mathrm{d}A$$

Suppose $R$ is bounded by a circle $C$ with a radius of $a$ centered at origin.  

$$ \int\!\int_R\mathrm{curl }\,\mathbf{F}\,\mathrm{d}A = 12\pi a^2 - 3\int\!\int_R r^2\,\mathrm{d}A$$

$$ 3\int\!\int_R r^2\,\mathrm{d}A = 3\int_0^{2\pi}\int_0^a r^3\,\mathrm{d}r\,\mathrm{d}\theta$$

$$ = \frac{3}{4}\int_0^{2\pi}a^4\,\mathrm{d}\theta = a^4\frac{3\pi}{2}$$

Thus,  

$$ \oint_C\mathbf{F}\cdot\mathrm{d}\mathbf{r} = 12\pi a^2 - a^4\frac{3\pi}{2}$$

Now, let  

$$ W:=W(a) = \oint_C\mathbf{F}\cdot\mathrm{d}\mathbf{r}$$

To find the max value of $W$, denoted as $W_\max$, we must find the roots of $W'(a)$.  

$$ W'(a) = 24\pi a - 6\pi a^3$$

$$ W'(a_e) = 0 \quad\implies\quad 4 - a_e^2 = 0  \quad\implies$$

$$ a_e^2 = 4 \quad\implies\quad a = 2$$

In [2]:
def dW(a): 
    return 24*np.pi*a - 6*np.pi*a**3

def W(a):
    return 12*np.pi*a**2 - (a**4)*3*np.pi/2

In [14]:
W(2)

75.39822368615503

Finally, $W=W_\max$ if and only if $C$ is a circle with a radius of $2$ centered at origin.  

$$ W_\max = W(2) = 48\pi - 24\pi = 24\pi$$

In [15]:
24*np.pi

75.39822368615503

**b)**  Compute this largest positive value.

As has been stated,  

$$ W_\max = W(2) = 24\pi \approxeq 75.4$$

### Problem 2

**a)**  $$ \mathbf{F} = \rho\mathbf{v} = \rho\,\frac{\mathrm{d}x}{\mathrm{d}t}\,\mathbf{i} + \rho \frac{\mathrm{d}y}{\mathrm{d}t}\,\mathbf{j} = \rho\langle x'(t),y'(t)\rangle$$

$$ \mathrm{div }\,\mathbf{F} = \frac{\partial \rho}{\partial x}\frac{\mathrm{d}x}{\mathrm{d}t} + \rho\frac{\partial x'(t)}{\partial x} + \frac{\partial \rho}{\partial y}\frac{\mathrm{d}y}{\mathrm{d}t} + \rho\frac{\partial y'(t)}{\partial y}$$

Integrating throughout the continuity equation over an arbitrary region $R$ bounded by $C$, a positively oriented simple closed curve, gives:  

$$ \int\!\int_R \frac{\partial \rho}{\partial t}\,\mathrm{d}A + \int\!\int_R\mathrm{div }\,\mathbf{F}\,\mathrm{d}A = 0$$

$$ \Phi := \Phi(t) = \oint_C \mathbf{F}\cdot\mathbf{n}\,\mathrm{d}s = \oint_C \rho v_x\,\mathrm{d}y - \rho v_y\,\mathrm{d}x$$

$$ \mathrm{div }\,\mathbf{F} = \frac{\partial \rho}{\partial x}\frac{\mathrm{d}x}{\mathrm{d}t} + \frac{\partial \rho}{\partial y}\frac{\mathrm{d}y}{\mathrm{d}t} = \frac{\mathrm{d}\rho}{\mathrm{d}t}$$

$$ \mathrm{d}\rho = \rho_x\,\mathrm{d}x + \rho_y\,\mathrm{d}y = \nabla \rho \cdot\mathrm{d}\mathbf{r}$$

where  

$$ \mathbf{r} := \mathbf{r}(t) = \langle x(t),y(t)\rangle $$

Thus,  

$$ \rho'(t) = \nabla \rho \cdot \mathbf{r}'(t) = \nabla \rho \cdot \mathbf{v}(t)$$

$$ \int\!\int_R\frac{\partial \rho}{\partial t}\,\mathrm{d}A = \int_a^b\int_{g_1(x)}^{g_2(x)}\partial \rho \,\frac{\mathrm{d}y\,\mathrm{d}x}{\partial t}$$

$$ = \int_a^b\rho\big(x,g_2(x),t\big)\,\frac{\mathrm{d}A}{\partial t} - \int_a^b\rho\big(x,g_1(x),t\big)\,\frac{\mathrm{d}A}{\partial t} $$